.vscodeignore

```diff
@@ -95,7 +95,7 @@ out/webviews/webview-side/viewers/ipywidgets/**
 out/webviews/webview-side/viewers/index.*.html
 out/webviews/webview-side/widgetTester/**
 out/pythonFiles/**
-!out/src/**
+out/src/**
```

Comment:
Restoring this line: https://github.com/microsoft/vscode-jupyter/blob/main/.vscodeignore#L97